terraform/modules/happy-env-eks/integration_secret.tf

```diff
@@ -11,8 +11,8 @@ locals {
     eks_cluster     = var.eks-cluster
     tags            = var.tags
     certificate_arn = module.cert.arn
-
-    ecrs = { for name, ecr in module.ecrs : name => { "url" : ecr.repository_url } }
+    ci_roles        = var.github_actions_roles
```

Comment:
TODO: make CLI function to read this so it easy for developers to get their CI role ARN

Reply:
https://github.com/chanzuckerberg/happy/pull/1686